mkdocs.yml

```diff
@@ -134,6 +134,11 @@ nav:
           - Optimistic concurrency control: develop/concepts/concurrency-control.md
           - Database transaction management: develop/concepts/transactions.md
   - Deploy: deploy/index.md
+  - How-tos:
```

Comment:
:thinking: On [L111](https://github.com/inveniosoftware/docs-invenio-rdm/pull/614/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R111) there is the "How-to guides" section already. On [L110](https://github.com/inveniosoftware/docs-invenio-rdm/pull/614/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R110) there is the "Topic guides/Administration Panel". So those could go in either of them. There is something to be said about future overall doc organization though because of the span of things we want to cover and how they are not necessarily apparent to someone coming to the docs for answers.

Reply:
> Do you think it's feasible to move all guides up to consolidate all guides into this new section location, and maybe categorize them under regular users and administrators/developers,

:+1: I think that makes sense. @kpsherva suggested something in a similar vein here: https://github.com/inveniosoftware/docs-invenio-rdm/issues/543 where the target audience is a bigger driver in the organization of the docs. I am not settled yet on what should be the top-level split: by features first and then audience or by audience first (which is sort of what we have but in verb form). Some experimentation here to be done for sure.